### ISSUE TITLE
nbtutor is not functioning properly

### ISSUE THREAD

**Issue by lgpage** [lgpage/nbtutor#32]
From https://groups.google.com/a/continuum.io/forum/#!topic/anaconda/45fNK2ni_so:
>(Windows +Mac OSx) When i run "%%nbtutor -r -f" the GUI does not display on the right hand side. I have installed nbtutor and no error message is displayed.
Thank you for the help.

**Comment by lgpage** [comment]
Please can you provide me with the following info:
- What browser are you using?
- The output from running the following commands in the terminal:
  - ``conda info``
  - ``conda list nbtutor``
  - ``conda list notebook``

**Comment by willingc** [comment]
@lgpage Looking at the info posted in the google group, it looks like the win32 system has Anaconda 64 installed.

**Comment by lgpage** [comment]
Is that even possible? To install the 64 version on a 32 machine?

**Comment by willingc** [comment]
Don't know?? I'm thinking that there is a configuration issue on the reporter's system.

**Comment by lgpage** [comment]
Hmm I know what the problem is. The ``py36_2`` build of ``v1.0.4`` is broken on conda-forge due to an update in the way jsplumb is packaged that I only caught after re-rendering the feedstock. Re-installing nbtutor to the ``py36_3`` build should fix this. My next task before anything else is to get some front-end tests in-place to make sure this doesn't happen again.

**Comment by Szepi** [comment]
Hmm, I have the same issue, using a Mac, using Python 3.5 (installed through port). I have followed the installation extension, including installing the extensions, see some proof below. Yet, the example provided does not work (no output on the right-side of the cell).
```bash
$ pip3 show nbtutor
Name: nbtutor
Version: 1.0.4
Summary: Visualize Python code execution in Jupyter Notebook cells
Home-page: https://github.com/lgpage/nbtutor
Author: Logan Page
Author-email: page.lg@gmail.com
License: BSD 3-Clause
Location: /opt/local/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages
Requires: notebook
$ jupyter nbextension list
Known nbextensions:
  config dir: /Users/csaba/.jupyter/nbconfig
    notebook section
      nbtutor/js/nbtutor.min  enabled
      - Validating: OK
  config dir: /opt/local/Library/Frameworks/Python.framework/Versions/3.5/etc/jupyter/nbconfig
    notebook section
      formgrade_extension disabled
      create_assignment/main  enabled
      - Validating: OK
    tree section
      formgrader/main  enabled
      - Validating: OK
      assignment_list/main  enabled
      - Validating: OK```

**Comment by lgpage** [comment]
@Szepi this seems to be related to issue #30

**Comment by Szepi** [comment]
True.

**Comment by lgpage** [comment]
Should be fixed by #44
